exercise-002/main.cpp

```diff
@@ -1,34 +1,68 @@
 #include <fmt/chrono.h>
 #include <fmt/format.h>
-
+# include <vector>
 #include "CLI/CLI.hpp"
 #include "config.h"
 
+
 auto main(int argc, char **argv) -> int
 {
     /**
      * CLI11 is a command line parser to add command line options
-     * More inpo at https://github.com/CLIUtils/CLI11#usage
+     * More info at https://github.com/CLIUtils/CLI11#usage
      */
     CLI::App app{PROJECT_NAME};
+    int count = 20;
     try
     {
+
         app.set_version_flag("-V,--version", fmt::format("{} {}", PROJECT_VER, PROJECT_BUILD_DATE));
+        app.add_option("-c,--count", count, "Variable count als Integer");
         app.parse(argc, argv);
+
     }
     catch (const CLI::ParseError &e)
     {
         return app.exit(e);
     }
+    std::vector<int> data(count);
+    fmt::print("Created a vector with {} elements\n", data.size());
+    
 
     /**
      * The {fmt} lib is a cross platform library for printing and formatting text
      * it is much more convenient than std::cout and printf
      * More info at https://fmt.dev/latest/api.html
      */
-    fmt::print("Hello, {}!\n", app.get_name());
+    fmt::print("Hello, {}!\n", count, app.get_name());
```

Comment:
```suggestion
    fmt::print("Hello, count: {} name: {}!\n", count, app.get_name());
```
Otherwise the name is omitted